tests/integration/targets/azure_rm_sqlserver/tasks/main.yml

```
@@ -47,6 +48,104 @@
       - output.changed == false
       - output.state == 'Ready'
 
+- name: Create extended instance of SQL Server
+  azure_rm_sqlserver:
+    resource_group: "{{ resource_group }}"
+    name: "sqlsrv-extended-{{ random_postfix }}"
```

The next step is to get the name and compare. Yaml treats "-" as a column when comparing, so it is recommended not to use "-" in the name. All of the following changes will be made. Thank you very much!
```suggestion
    name: "sqlsrvextended{{ random_postfix }}"
```

---

Can you please elaborate on why you don't want dashes in the name? It is valid YAML and there are existing test cases which do the same. Keeping the dashes improves visibility so I would prefer them to stay.